### DISCUSSION TITLE
Délai supplémentaire pour le labo 5

### DISCUSSION THREAD

**Discussion by cgerro** [web-classroom/web-classroom.github.io#12] [Q&A : Questions organisationnelles]
Bonjour,
Je parle au nom des personnes qui sont en sécurité mais peut-être que les logiciels se sentiront aussi concernés par ma demande :)
Nous sommes à un moment du semestre où nous avons beaucoup de travail dans la plupart des branches et nous avons notamment un midterm dans le cours de CAA (Cryptographie avancé) mercredi prochain. 
Afin de pouvoir réaliser le laboratoire actuel dans des bonnes conditions et surtout de pouvoir le finir, serait-il possible d'avoir un délai supplémentaire pour le rendu ? 
Bonne suite de journée.

**Comment by OlivierLmr** [comment]
Bonjour,

Je vois effectivement qu'une grande partie de la classe n'a pas encore fait beaucoup de progrès sur ce labo.

Étant donné que nous sommes en avance (ceci impliquant donc qu'il s'agit d'un cas particulier et exceptionnel, d'une coïncidence avantageuse), disons effectivement que vous avez 2 semaines pour ce labo-ci. Il sera donc à rendre pour le 14 novembre, au lieu du 7.

J'en profite pour rappeler à tout le monde de _me poser vos questions si vous êtes bloqués_, le but étant de perdre le moins de temps possible.

Bonne chance pour l'examen, et à mardi prochain.

**Reply by cgerro** [reply]
Bonjour, merci beaucoup pour le délai ! 
Bon week-end.
